### ISSUE TITLE
Should ClipboardItem be marked as [SecureContext]?

### ISSUE THREAD

**Issue by whsieh** [w3c/clipboard-apis#103]
Both `Clipboard` and `navigator.clipboard` are limited to secure contexts, but `ClipboardItem` is not. Is there any value in exposing `ClipboardItem` in non-secure contexts in the absence of any instance of `Clipboard`, or should be additionally limit `ClipboardItem` to secure contexts only?

**Comment by pwnall** [comment]
Good catch!

I agree. I think it'd be more consistent to have `[SecureContext]` on both `ClipboardItem` and `ClipboardEvent`.

**Comment by snianu** [comment]
@pwnall @whsieh I agree with `ClipboardItem`, but not sure about `ClipboardEvent` as it contains the legacy `DataTransfer` API and changing this could have backward compat problems.

**Comment by EdgarChen** [comment]
Shall we close this as ClipboardItem has marked as [SecureContext]?

**Comment by snianu** [comment]
Yep, I think we can close this issue.
